resources/var/www/html/styles/default.css

```
@@ -0,0 +1,37 @@
+body {
```

```suggestion
:root {
```
use the root-element instead of body